kso_utils/project.py

```diff
@@ -1354,7 +1350,7 @@ def __init__(self, *args, **kwargs):
         self.modules = g_utils.import_modules(["zenodo_utils"], utils=True)
         self.modules.update(
             g_utils.import_modules(
-                ["torch", "wandb", "yaml", "ultralytics"],
+                ["torch", "wandb", "yaml", "ultralytics", "pims", "itertools", "mlflow"],
```

Comment:
Not a big deal, but for next time: these kind of patterns and commits (making something dynamic because only used in one class of the file) point in another direction: maybe the classes should not be in the same file! Moving MLProjectProcessor to another file should generally be no issue, and then we wouldn't have to have all these dynamic imports. I think we should probably still fix that